Modules/Scripted/EditorLib/HelperBox.py

```diff
@@ -165,19 +165,17 @@ def setVolumes(self,masterVolume,mergeVolume):
 
   def setMasterVolume(self,masterVolume):
     """select merge volume"""
+    if isinstance(masterVolume, basestring):
+      masterVolume = slicer.mrmlScene.GetNodeByID(masterVolume)
     self.masterSelector.setCurrentNode( masterVolume )
-    self.select()
```

Comment:
There's some tricky corner cases around switching master volumes and bringing up corresponding merge and per-structure label volumes.  @fedorov pointed out an issue lately where this was problematic when controlling the editor from another module.  We should look out for any issues that fall out from changing this logic.